guides/advanced_index_actions.md

```diff
@@ -0,0 +1,117 @@
+# Advanced Index Actions Guide
+- [Advanced Index Actions](#advanced-index-actions)
+  - [Setup](#setup)
+  - [Api Actions](#api-actions)
+    - [Clear Index Cache](#clear-index-cache)
+    - [Flush Index](#flush-index)
+    - [Refresh Index](#refresh-index)
+    - [Open/Close Index](#open-close-index)
```

Comment:
This link is not working.

Reply:
@dblock This guide has broken linking because of the use of `[Open/Close Index](#open-close-index)` syntax. I think the `/` character is breaking the` #open-close-index` link, but not sure. Regardless, I need to change the section title to use a `Open or close index` & make the hash link `#open-or-close-index`.(or something similar)

* This isn't an issue in the Ruby guide version because those guides don't include the MD hash linking at the top of the [guide](https://raw.githubusercontent.com/opensearch-project/opensearch-ruby/main/guides/advanced_index_actions.md)

Is the section title I suggested okay to you? Or would you prefer some other title/hash-link?

Reply:
Perfectly fine!